js/modules.js

```diff
@@ -11,17 +11,23 @@ module.exports = (ENV, { MODULES, THIRDPARTY }) => {
   return {
     rules: [
       {
-        // .js and .jsx files are caught
-        test: /\.jsx?$/,
+        test: /\.m?js/,
```

Comment:
There's some newish module JS format that ship content with `*.mjs` file. That was really annoying webpack. So I had to try to hack some config in here to handle it.